docs/user/introduction.md

```diff
@@ -1,56 +1,104 @@
-that are collections of one or more _Node-RED instances_. Each instance is created
-from a _Template_ to provide its default settings and runs on a _Stack_ that determines
-the Node-RED version, memory and CPU usage.
+1. **Automatic Creation**: Your first Node-RED instance should be automatically created upon your initial login to FlowFuse. To access the Node-RED Editor, simply click on `Open Editor`.
 
-_Devices_ can be assigned to an instance and have flows automatically deployed to
-them when _snapshots_ are created of the instance.
+    ![Open Editor](./images/getting-started/Open-Editor.png)
 
-For more details on these and other core concepts, you can learn about them [here](concepts.md).
+2. **Additional Instance**: For utilizing various other FlowFuse features (e.g., DevOps Pipelines), it's highly beneficial to create a second Node-RED instance. A second Node-RED instance is included in both our Starter Tier and the Trial Phase of FlowFuse Cloud. To do so, select your application— in our example, "Demo's Application"— and click `Add Instance`.
 
-## Working with Applications
+    ![Add Instance](./images/getting-started/Add-Instance.png)
 
- - [DevOps Pipelines](devops-pipelines.md) - How to create DevOps pipelines to easily deploy between Node-RED instances
+[Learn more about Instances](#working-with-instances)
 
-## Working with Instances
+## Creating Your First Flow
 
- - [Snapshots](snapshots.md) - Create point-in-time backups of your Node-RED instances.
- - [Environment Variables](envvar.md) - How to manage Environment Variables in your Node-RED instances.
- - [Change Project Stack](changestack.md) - How to change an instance stack, for example to upgrade Node-RED.
- - [Logs](logs.md) - The Logs available in the FlowFuse application.
- - [Project Link Nodes](projectnodes.md) - Custom nodes for sending messages between Node-RED instances and devices.
- - [Instance Settings](instance-settings.md) - Settings available for Node-RED instances.
- - [Shared Team Library](shared-library.md) - Share flows easily between different Node-RED instances in your team.
- - [Node-RED Tools Plugin](/docs/migration/node-red-tools.md) - A plugin for Node-RED that lets you work with your flows outside of FlowFuse.
- - [High Availability mode](high-availability.md) - Run multiple copies of your instance for scaling and availability.
+If you are already familiar with Node-RED, you can [skip this section](#creating-your-first-devops-pipeline).
 
-## Working with Teams
+1. **Access the Node-RED Editor**.
 
- - [Team management](./team/) - How to add and remove users from a team.
- - [Role based access control](./team/#role-based-access-control) - Which privileges are granted to different roles.
+2. **Add an Inject Node**: The Inject node allows you to manually or automatically inject messages into a flow. Drag one onto the workspace from the palette. After adding, you can review its properties in the Information sidebar.
```

Comment:
I think this is better explained by robs getting started article: https://flowfuse.com/blog/2023/01/getting-started-with-node-red/#first-flow

Should we link, or update that post? Or is there another way we can make sure we have one resource to get started?

Reply:
Good catch, I'll update the doc